assignments/objects.js

```diff
@@ -19,27 +19,84 @@ const example = {
 
 // Write your intern objects here:
 
+const internMitzi = {
+  "id": 1,
+  "first name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+};
+
+const internKennan = {
+  "id": 2,
+  "first name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M",
+};
+
+const internKeven = {
+  "id": 3,
+  "first name": "Keven",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"
+};
+
+const internGannie = {
+  "id": 4,
+  "first name": "Gannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M"
+};
+
+const internAntonietta = {
+  "id": 5,
+  "first name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F"
+};
+
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
 
+console.log(internMitzi["first name"]);
+
 // Kennan's ID
 
+console.log(internKeven["id"]);
```

Comment:
internKeven.id would also work just on the offchance you didn't know.  Dot notation for a single word is valid.  Anything with a number or space needs to be referenced with bracket notation.